gdcdictionary/schemas/molecular_test.yaml

```diff
@@ -892,6 +915,17 @@ properties:
       - Unknown
       - Not Reported
       - Not Applicable
+      - MIB1
+      - MAML3 (HCMI)
```

Comment:
Is the "HCMI" in DAT-2741 just someone denoting who asked for this gene @tlicht3 ?

Reply:
yes - that was an obvious fix - I will make sure this is updated.